src/amo/pages/Block/index.js

```diff
@@ -140,15 +114,40 @@ export class BlockBase extends React.Component<InternalProps> {
       return <ServerErrorPage />;
     }
 
-    const title =
-      block && block.name
-        ? i18n.sprintf(
-            i18n.gettext(`%(addonName)s has been blocked for your protection.`),
-            {
-              addonName: block.name,
-            },
-          )
-        : i18n.gettext(`This add-on has been blocked for your protection.`);
+    const isSoftBlocked =
+      block?.soft_blocked.length &&
+      (block?.soft_blocked.includes(match.params.versionId) ||
+        !block?.blocked.length);
+    let title;
```

Comment:
We specifically dropped the "Versions blocked" text partly because it could be confusing, I fear this could have similar issues: if you show the version number alone, it doesn't say anything about the other versions, and that could be misleading ? I've asked in the Add-ons UX slack channel though, and I'll file and implement a follow-up if we decide we want the version in the title/headline.